### PR TITLE
feat: granular thorchain savers deposit and withdraw flags

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - ~~note, swapper will follow this in the stack.~~

Disregard the above - no swapper to follow, as this should already be handled at quotes level (failure to get a quote) and/or halt status (we already check for active THORChain pool status for swapper)

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/6520

> What protocols, transaction types or contract interactions might be affected by this PR?

Low - this should come as no change for THORCHain savers

### Engineering

- THORChain savers pools are shown as disabled with the flags off in the overview modal

### Operations

- THORChain savers pools show no regressions against develop

## Screenshots (if applicable)

- Savers deposits and withdraws disabled

Screenshot 2024-03-21 at 23 43 24
Screenshot 2024-03-21 at 23 43 19

- Savers deposits and withdraws enabled

Screenshot 2024-03-21 at 23 47 29

### CODE REVIEW

**Comment by gomesalexandre** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/shapeshift/web/6535?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#6535** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6535?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#6532** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6532?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6527** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6527?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx [hunk 1]
```diff
@@ -244,23 +250,24 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
...
-        isDisabled: hasPendingTxs || hasPendingQueries,
-        toolTip:
-          hasPendingTxs || hasPendingQueries
-            ? translate('defi.modals.saversVaults.cannotWithdrawWhilePendingTx')
-            : undefined,
+        isDisabled: hasPendingTxs || hasPendingQueries || isHaltedWithdrawals,
+        toolTip: (() => {
+          if (hasPendingTxs || hasPendingQueries)
+            return translate('defi.modals.saversVaults.cannotWithdrawWhilePendingTx')
+          if (isHaltedWithdrawals) return translate('defi.modals.saversVaults.haltedWithdrawTitle')
```
Comment (kaladinlight) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx:269]: feels like halted should have tool tip priority here

File: src/assets/translations/en/main.json [hunk 2]
```diff
@@ -373,6 +373,7 @@
         "vaultCap": "Vault Cap",
         "vaultCapTooltip": "Current maximum capacity for this vault",
         "haltedTitle": "Deposits are temporarily halted.",
```
Comment (kaladinlight) [inline src/assets/translations/en/main.json:375]: nitpick rename suggestion
```suggestion
        "haltedDepositTitle": "Deposits are temporarily halted.",
```

Comment (gomesalexandre) [reply src/assets/translations/en/main.json]: 2c51df55d6607a077137e8b3a8fc3679a67fe845

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**